java/ActigraphReader.java

```diff
@@ -104,12 +104,12 @@ else if (tokens[0].trim().equals("Stop Date"))
                         else if (tokens[0].trim().equals("Serial Number"))
                             serialNumber=tokens[1].trim();
                         else if (tokens[0].trim().equals("TimeZone"))
-                            infoTimeZone=tokens[1].trim();
+                            infoTimeShift=tokens[1].trim(); // gt3x calls time shift as time zone
                     }
                 }
             }
 
-            System.out.println("Timezone: " + infoTimeZone);
+            System.out.println("Timezone: " + infoTimeShift);
```

Comment:
What do you think about making this more explicit, like "Time shift wrt. local" or just "Time shift" instead of "Timezone"? I'm thinking in a few years time when our memories have faded it would be less confusing for future developers/phd students/postdocs :student:

Reply:
Hmm this is really just printing out the TimeZone field from `info.txt` because that's how it is written. So not sure if we should change this.

Reply:
Personally would prefer not to perpetuate the confusing manufacturer's decision making. Since this print out is exposed to the end user I think it would be better to just print time shift. @R-Walmsley @aidendoherty what do you think about this? Context: GT3X's internal bookkeeping has a variable 'TimeZone' but it is actually a time shift wrt. the local time.

Reply:
Come to think of it, there's also a `--timeZone` flag (default Europe/London) in accProcess.py which would add to the confusion. Perhaps a compromise would be to print "Device timezone" but again I'd prefer "Device timeshift"

Reply:
Yeah maybe it is confusing. From a user perspective actually, it would be odd to change the name of the extracted field because all else have the same field names except for the time zone.  Other software packages also used time zone https://github.com/THLfi/read.gt3x/blob/c4b83cc262c7b3dc66b0408f9c72123686dd3e33/R/readGT3X.R#L187

Maybe just to avoid confusion with two time zones being the same thing, we shall use `device timezone`.

Reply:
It's so confusing in more than one way :sweat_smile: I read this [opinionated blog](https://spin.atomicobject.com/2016/07/06/time-zones-offsets/#:~:text=A%20time%20zone%20is%20a,because%20of%20Daylight%20Saving%20Time.) some time ago when I was fixing the Axivity timestamps, and read it again today. Timezones are `"Europe/London"`, `"GMT"`, etc. and time _offsets_ are `"+01:00"`, `"-09:00"`, etc. but people usually mix them up. They shouldn't because a single timezone can have multiple offsets (like Europe/London which is +01 in Apr-Oct and +00 in Oct-Apr).

Adding to the confusion is the fact that, as you know, the device's offset is really only accurate at instant t=0 :sweat_smile: because any time after that can suffer a crossover and now we are in a different offset. That's why I wasn't tempted to call it _offset_ (nor timezone), just "timeshift". To be absolutely strict one would have to print the more verbose `"Device's initial offset"`

I don't think it would be that odd to change the name, as we already do inside the code. Are users likely to examine info.txt?

Reply:
Hi Hang, what's your final opinion on this? My preference is that we print `Device's initial offset` instead of `Timezone` (which might be confused with the user-inputted timezone, e.g. `Europe/London`.